tool/tctl/common/token_command.go

```diff
@@ -175,6 +176,27 @@ func (c *TokenCommand) Add(client auth.ClientI) error {
 		return trace.BadParameter("this cluster has no auth servers")
 	}
 
+	// Print signup message formatted with JSON or just print the raw token.
+	if c.format == teleport.JSON {
+		expires := time.Now().Add(c.ttl)
+		j := map[string]interface{}{
+			"token":   token,
+			"roles":   roles,
+			"expires": expires,
+		}
+
+		data, err := json.MarshalIndent(j, "", " ")
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		fmt.Print(string(data))
+
+		return nil
+	} else if c.format == teleport.Token {
+		fmt.Println(token)
```

Comment:
Why do we need a new format for "Token"? Can you just use `teleport.Text` here? It _is_ text, after all.

Reply:
@r0mant recommended that I add this format to print the raw token, making it easy to pass though unix pipelines without having to decode json

Reply:
Yeah I think it's useful to have format that prints only token. Alternatively, we could use `text` format to print just the token and output the standard full message if format is empty.

Reply:
But you can print the raw token with format "text" - right?

Reply:
No, `text` currently prints a user-friendly message that's not easily machine readable. 

@r0mant that makes sense to me, but it seems tctl and tsh both use `text` formatting to mean a user-friendly message. So probably don't want to make that inconsistent right?

Reply:
Yeah, I think that historically `text` format in our case means "human-friendly" or "table" output. Which is why I suggested `--format=token`. I don't think it's a big deal, but let's double-check with @xinding33 just in case.

@xinding33 - Are you ok with having a `--format=token` flag for the `tctl token add` command that outputs _only_ token value? By default the command outputs the verbose message which users have to use bash CLI magic to parse out the token which is cumbersome.

Reply:
Now I see.

If anything, I would recommend `--format=raw`, so that we can reuse it for things that are not tokens.

Reply:
Yeah, or maybe `--format=short`? What's the antonym of "verbose"? :)

Reply:
If I understand this correctly, we are talking about supporting the following 5 formats:

1. json
2. yaml
3. text
4. token
5. pty

It's a lot of formats and we are also introducing a new one in `token` that's not used anywhere else in the product. Do we plan to use this format elsewhere in the product in the future? If not, then I like @r0mant's suggestion of using `text` format to print just the token and outputting the standard full message if format is empty.